contracts/StakedCelo.sol

```diff
@@ -46,4 +66,64 @@ contract StakedCelo is ERC20Upgradeable, UUPSOwnableUpgradeable, Managed {
     function burn(address from, uint256 amount) external onlyManager {
         _burn(from, amount);
     }
+
+    /**
+     * @notice Locks vote stCELO from an address.
+     * @param account The address that will have its stCELO balance locked.
+     * @param amount The amount of stCELO to lock.
+     */
+    function lockVoteBalance(address account, uint256 amount) external onlyManager {
+        uint256 lockedBalance = _lockedBalances[account];
+        if (lockedBalance < amount) {
+            _lockedBalances[account] = amount;
+            uint256 amountToSubtract = amount - lockedBalance;
+            uint256 accountBalance = balanceOf(account);
+            require(accountBalance >= amountToSubtract, "Not enough locked stCelo");
+            unchecked {
+                _balances[account] = accountBalance - amountToSubtract;
+            }
+            emit LockedStCelo(account, amount);
+        }
+    }
+
+    /**
+     * @notice Returns vote stCELO locked balance.
+     * @param account The address of locked stCELO balance.
+     * @return The amount of locked stCELO.
+     */
+    function lockedVoteBalanceOf(address account) public view returns (uint256) {
+        return _lockedBalances[account];
+    }
+
+    /**
+     * @notice Unlocks vote stCELO from an address.
+     * @param beneficiary The address that will have its stCELO balance unlocked.
+     */
+    function unlockVoteBalance(address beneficiary) public {
+        uint256 previouslyLocked = _lockedBalances[beneficiary];
+        require(previouslyLocked > 0, "No locked stCelo");
+        uint256 currentlyLocked = IManager(manager).updateHistoryAndReturnLockedStCeloInVoting(
+            beneficiary
+        );
+        require(previouslyLocked > currentlyLocked, "Nothing to unlock");
```

Comment:
>=?

Reply:
If previouslyLocked = currently locked -> there is nothing to unlock 

">" is ok